src/app/components/homepage/carousel/carousel.component.css

```diff
@@ -0,0 +1,14 @@
+img{
+    height: 450px;
+    width: 992px;
```

Comment:
this will not be responsive - CSS has different ways of making images scale correctly based on the breakpoint - check online and rewrite this